workspace-extension/src/extension.ts

```diff
@@ -104,15 +146,14 @@ export function activate(context: vscode.ExtensionContext) {
         let cleanup = new Array<{ (): void }>(() => proxy.close());
         try {
           let message = `Proxy to ${target.host}:${target.port}/udp listening on ${proxy.listenPort}/tcp`;
-          console.log(message);
```

Comment:
Curious about this removal. Is it important? If its better to not log, should we remove the `message` variable?

Reply:
No idea, I don't know where the logs show up so I didn't see the point in logging. Should probably remove message as well yeah.

Reply:
The "Output" in vscode has a dropdown on right where logs from extensions are shown. I _think_ this shows up there. I'm fine with deleting it regardless and I'm removing the `let message` in my finishing up of this PR